### ISSUE TITLE
`contentDirExclude` doesn't exclude anything

### ISSUE THREAD

**Issue by Xenfo** [contentlayerdev/contentlayer#291]
```ts
export default makeSource({
  contentDirPath: 'data',
  contentDirExclude: ['themes'],
})
```
```
Warning: Found 1 problems in 8 documents.

 └── Couldn't find document type definitions provided by name for 1 documents. (Skipping documents)
     
     Please use one of the following document type names: Post, Project.
     
     • themes/some-vscode-theme.json (Used type name: "dark")
```

Why is it still trying to load the JSON file?

**Comment by schickling** [comment]
Thanks for opening this issue. Can you please provide a reproduction repo with instructions how to reproduce the problem you're experiencing? Thank you!

**Comment by Xenfo** [comment]
https://github.com/Xenfo/website-repro

**Comment by schickling** [comment]
Thanks a lot. This will be fixed in the upcoming 0.2.8 release.
